test/models/post_image_test.rb

```diff
@@ -0,0 +1,23 @@
+# == Schema Information
+#
+# Table name: post_images
+#
+#  id         :integer          not null, primary key
+#  user_id    :integer
+#  image      :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_post_images_on_created_at  (created_at)
+#  index_post_images_on_user_id     (user_id)
+#
+
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.